eleventy.config.js

```diff
@@ -43,12 +40,7 @@ module.exports = function (config) {
         return md.render(content);
     });
 
-    var markdownItLibrary = md.use(require("markdown-it-anchor"), { slugify: slug });
-
-    config.setLibrary("md", markdownItLibrary);
```

Comment:
Rather than deleting this, which will break things, you'll need to follow these instructions to modify the default Markdown configuration in `eleventy-plugin-fluid`: https://github.com/fluid-project/eleventy-plugin-fluid#markdown-configuration

Reply:
@greatislander, when I follow the instruction and updated eleventy-plugin-fluid's config, but build fails after that with errors. This is the changes I made for the markdown option, I commented it for now.

```
markdown: {
    options: {
        slugify: slug
    },
    plugins: [
        markdownItAnchor
    ]
},
```